hrms/hr/doctype/employee_separation_template/employee_separation_template.json

```diff
@@ -282,10 +315,11 @@
  "quick_entry": 0, 
  "read_only": 0, 
  "read_only_onload": 0, 
- "show_name_in_global_search": 0, 
+ "show_name_in_global_search": 0,
+ "show_title_field_in_link": 1, 
  "sort_field": "modified", 
  "sort_order": "DESC", 
- "title_field": "designation", 
+ "title_field": "title", 
```

Comment:
This will also need a patch to set the current title field = designation in all the existing onboarding & separation templates

Reply:
All the existing onboarding & separation templates will display `ID/Name(Primary Key)`, in `Title` column  if it is not set.

Reply:
If we are to write a patch will it be under v15 or v14?

Reply:
> All the existing onboarding & separation templates will display ID/Name(Primary Key), in Title column if it is not set.

It will display in the list view but the title column will be empty for all the existing records. And for those who are used to seeing designation in the list view will see IDs and can get confused. 

People will have to manually go and update each record. Also if they want to set some other field, the title field will throw a mandatory error.

> If we are to write a patch will it be under v15 or v14?

We can port this to v14, so v14 folder